modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -17,6 +21,45 @@ def show
           )
         end
 
+        def appoint_individual
```

Comment:
Unless I'm completely missing something, I think there has to be some level of authorization check here to make sure the person making the request is actually authorized to do so.  It seems like we would only want to allow the Veteran themself to actually use this endpoint.  (again, I could totally be missing something)

Reply:
Yup, forgot to wire that in. [Added](https://github.com/department-of-veterans-affairs/vets-api/pull/8024/commits/f48bb955aaf2b76b17b980fbb68fee1c841656aa). Will probably make it a `before-` action validation once there's more of them.

Reply:
actually, realized that `show` is just re-implementing `verify_access!`, so I will move this to a top level controller thing